distribution/std-lib/Standard/src/Image/Data/Matrix.enso

```diff
@@ -0,0 +1,150 @@
+from Standard.Base import all
+import Standard.Image.Data.Histogram
+import Standard.Image.Data.Image
+import Standard.Image.Data.Matrix.Internal
+
+polyglot java import org.enso.image.data.Matrix as Java_Matrix
+
+type Index_Out_Of_Bounds_Error index
```

Comment:
The type needs documentation, and you also need to define a pretty-printed `.to_display_text` so it shows up nicely in the IDE.